graphql/schemabuilder/reflect_test.go

```diff
@@ -210,20 +226,65 @@ func TestEnumMapWrongArg(t *testing.T) {
 
 }
 
-func TestEnumMapKeys(t *testing.T) {
+func TestEnumMapWrongArg2(t *testing.T) {
```

Comment:
Let's make the test names more descriptive to the case, i.e.`TestEnumMapMismatchedValues`